tensorflow/lite/micro/tools/make/targets/bluepill_makefile.inc

```diff
@@ -58,12 +54,24 @@ ifeq ($(TARGET), bluepill)
     $(MAKEFILE_DIR)/downloads/stm32_bare_lib/source/debug_log.c
   MICROLITE_CC_SRCS := $(filter-out $(EXCLUDED_SRCS), $(MICROLITE_CC_SRCS))
   TEST_SCRIPT := tensorflow/lite/micro/testing/test_bluepill_binary.sh
-  # These are tests that don't currently work on the blue pill.
+
+  # TODO(b/143286954): Figure out why some tests fail and enable ince the issues
+  # are resolved.
   EXCLUDED_TESTS := \
     tensorflow/lite/micro/micro_interpreter_test.cc \
-    tensorflow/lite/micro/simple_tensor_allocator_test.cc
+    tensorflow/lite/micro/micro_allocator_test.cc \
```

Comment:
I assume these tests never worked for bluepill?

Reply:
not in the recent past. I have an internal bug to track this down.